pipelines/e2e/e2e.yaml

```
@@ -36,6 +36,10 @@ spec:
       type: string
       description: Public key to use for validation by the enterprise contract
       default: k8s://openshift-pipelines/public-key
+    - name: enterpriseContractExtraRuleData
+      type: string
+      description: Extra rule data to be merged into existing policy data. Use syntax "key1=value1,key2=value2..."
```

Thinking about the description, maybe this is helpful to explain:

```suggestion
      description: Extra rule data to be merged into the policy specified in above. Use syntax "key1=value1,key2=value2..."
```

or

```suggestion
      description: Extra rule data to be merged into the policy specified in params.enterpriseContractPolicy. Use syntax "key1=value1,key2=value2..."
```

---

@simonbaird I like the 2nd option better, added.